### ISSUE TITLE
Use of BigInt64Array and BigUint64Array breaks older browsers and iOS Safari

### ISSUE THREAD

**Issue by florianf** [fabiospampinato/proxy-watcher#1]
Those two classes are undefined, maybe check if they are available first?

See: https://caniuse.com/#search=BigInt64Array

Stacktrace: 
```
ReferenceError: BigInt64Array is not defined
isTypedArray utils.js:25
cloneCustomizer utils.js:15
baseClone lodash.js:2627
cloneDeepWith lodash.js:11121
cloneDeep utils.js:13
....
store.js:17
```

Thanks for your great store implementation, really enjoying it so far!

**Comment by fabiospampinato** [comment]
👍 Do you know if there are any other objects that may not be implemented in environments that implement Proxy?

**Comment by florianf** [comment]
It seems 
BigInt64Array, https://caniuse.com/#search=BigInt64Array
BigUint64Array, https://caniuse.com/#search=BigUint64Array
are the only culprits.

Float64Array https://caniuse.com/#search=Float64Array seems to be supported well.

Thx!

**Comment by fabiospampinato** [comment]
Fixed in v1.2.1.

Let me know if you find any other issues and/or have any suggestions regarding this library or Store.
